jib-core/src/integration-test/java/com/google/cloud/tools/jib/registry/AuthenticationMethodRetrieverIntegrationTest.java

```diff
@@ -26,7 +26,7 @@
 /** Integration tests for {@link AuthenticationMethodRetriever}. */
 public class AuthenticationMethodRetrieverIntegrationTest {
 
-  private static final EventHandlers EVENT_HANDLERS = EventHandlers.none();
+  private static final EventHandlers EVENT_HANDLERS = EventHandlers.NONE;
```

Comment:
Maybe at this point, we don't need the field (`EVENT_HANDLERS`) anymore. We can use `EventHandlers.NONE` directly in code.